shared/lib_battery_lifetime_calendar_cycle.cpp

```diff
@@ -210,6 +212,66 @@ double lifetime_cycle_t::average_range() { return state->average_range; }
 
 double lifetime_cycle_t::capacity_percent() { return state->cycle->q_relative_cycle; }
 
+void lifetime_cycle_t::resetDailyCycles() {
```

Comment:
I like that this code has moved into the cycle model, good thinking!

Reply:
Thanks!